extensions/VisualEditor/wikia/modules/ve/ve.track.js

```diff
@@ -37,9 +37,13 @@ require( ['wikia.tracker'], function ( tracker ) {
 				};
 			},
 			'mwtiming.performance.system.activation': function ( data ) {
+				// INT-322 A/B test: Image / Video upload for toolbar
+				var label = (ve.init.wikia.ToolbarABTestVariantNumber() === 1) ?
```

Comment:
It's ok to generate new event (another one), but the original one should still be generated. In the future we will forget about the experiment and we will be curious why we had sudden dive and then come back in "edit-page-ready" events.